app/templates/client/ng1/dev/app.config.js

```diff
@@ -5,11 +5,9 @@
     .config([
       "$locationProvider",
       function($locationProvider) {
-        <% if (!cordova) { %>
-        $locationProvider.html5Mode(true);
-        <% } else {%>
-        $locationProvider.html5Mode(true);
```

Comment:
I know it was wrong, but didn't cordova need the `html5Mode` to be set to `false`? Because of all the routing problems.